__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+
+const db = require('../db/connection.js')
+const testData = require('../db/data/test-data')
+const request = require('supertest')
+const app = require('../app.js')
+const seed = require('../db/seeds/seed')
+
+
+beforeEach(() => 
+    seed(testData)
+);
+
+afterAll(()=> 
```

Comment:
If you are getting any jestTimeout errors, refactor this afterAll so that you are not returning db.end()